File: RaizenUserRegister.Domain/Response/UserResponse.cs
namespace RaizenUserRegister.Domain.Response
{
    public class UserResponse
    {
        public string id { get; set; }
        public string email { get; set; }
        public string name { get; set; }
        public string secondName { get; set; }
        public string document { get; set; }
        public string nickname { get; set; }
        public string password { get; set; }

        public AddressResponse address { get; set; }
    }
    
}

File: presentation/src/model/userResponse.ts
export class UserResponse {
  id: string = "";
  email: string = "";
  name: string = "";
  secondName: string = "";
  document: string = "";
  nickname: string = "";
  password: string = "";
  address: Address = new Address();
}
export class Address {
  logradouro: string = "";
  bairro: string = "";
  uf: string = "";
  localidade: string = "";
  city: string = "";
  cep: string = "";
}
